apps/heft/src/pluginFramework/HeftTaskSession.ts

```diff
@@ -164,55 +165,33 @@ export interface IChangedFileState {
 }
 
 /**
- * Options that are used when globbing the set of changed files.
+ * Used to specify a selection of files to copy from a specific source folder to one
+ * or more destination folders.
  *
  * @public
  */
-export interface IGlobChangedFilesOptions {
+export interface IIncrementalCopyOperation extends ICopyOperation {
   /**
-   * Current working directory that the glob pattern will be applied to.
+   * If true, the file will be copied only if the source file has changed since it was last copied.
```

Comment:
If true, the copy operation will only consider files that have changed since the last time the current task executed.